x-pack/test_serverless/shared/services/bsearch_secure.ts

```diff
@@ -25,7 +25,7 @@ const parseBfetchResponse = (resp: request.Response): Array<Record<string, any>>
 };
 
 interface SendOptions {
-  supertestWithoutAuth: SuperTest.SuperTest<SuperTest.Test>;
+  supertestWithoutAuth: SuperTest.Agent;
```

Comment:
@pgayvallet Hey man, should this type differ from the regular super test, with auth?

Reply:
Not really, the type is the same for the 2 instances